debian/tests/test

```diff
@@ -0,0 +1,23 @@
+#!/bin/sh
+
+set -ex
+
+DIR_NAME=$(dirname "$0")
+SCRIPT_NAME=$(basename "$0")
+
+# Source the environment variables from the .env file
+. "$DIR_NAME"/../../.env
+
+if [ "$SCRIPT_NAME" = "test" ]; then
```

Comment:
A little bit too much black magic I think with the 2 test fails. Maybe it will be simpler to have one test file with some arguments? like with-sudo/no-sudo?

That way we keep the one single script to run them all, but it’s a little bit more explicit.